infrastructure/docker/Dockerfile-repository-server

```diff
@@ -0,0 +1,15 @@
+FROM registry.fedoraproject.org/f33/python3
```

Comment:
Should we pin to a specific digest/tag?

Reply:
Yes, it should be better.   I'll put it.  Thanks!